INSTALLING.md

```diff
@@ -20,11 +20,16 @@ Note the current compatability matrix:
 
 ---
 
-### Step 2. System-level dependencies
+### Step 2. System-level dependencies and Clang-17
```

Comment:
don't think we need to add clang-17 here

Reply:
done